operator/pkg/central/values/translation/translation_test.go

```diff
@@ -609,8 +788,14 @@ func TestTranslate(t *testing.T) {
 		t.Run(name, func(t *testing.T) {
 			wantAsValues, err := translation.ToHelmValues(tt.want)
 			require.NoError(t, err, "error in test specification: cannot translate `want` specification to Helm values")
+			var allExisting []ctrlClient.Object
+			for _, existingPVC := range tt.args.pvcs {
+				allExisting = append(allExisting, existingPVC)
+			}
+			client := fkClient.NewClientBuilder().WithObjects(allExisting...).Build()
+			translator := Translator{Client: client}
 
-			got, err := translate(tt.args.c)
+			got, err := translator.translate(context.TODO(), tt.args.c)
```

Comment:
```suggestion
			got, err := translator.translate(context.Background(), tt.args.c)
```
Tests are one of the canonical uses of `Background`, FWIW.